android/app/src/main/res/values-es/strings.xml

```diff
@@ -923,7 +923,7 @@
 	<string name="type.amenity.vending_machine.drinks">Máquina expendedora de bebidas</string>
 	<string name="type.amenity.vending_machine.food">Máquina expendedora de comida</string>
 	<string name="type.amenity.vending_machine.newspapers">Máquina expendedora de periódicos</string>
-	<string name="type.amenity.vending_machine.parking_tickets">Máquina de pago de tique de aparcamiento</string>
+	<string name="type.amenity.vending_machine.parking_tickets">Tique de aparcamiento</string>
```

Comment:
Is this what is shown or just meta data for search? Because if "Máquina" is removed, then it isn't entirely correct. Maybe we can use "Central de pago de aparcamiento" ?